plugin/mutate/source/dextool/plugin/mutate/backend/database/schema.d

```diff
@@ -411,6 +412,33 @@ struct MutantTimeoutCtxTbl {
     State state;
 }
 
+import dextool.plugin.mutate.backend.database.type : Rationale;
+
+@TableName(markedMutantTable)
+@TablePrimaryKey("st_id")
+struct MarkedMutantTbl {
+    @ColumnName("st_id")
+    long mutationStatusId;
+
+    long id;
+
+    uint line;
+
+    uint column;
+
+    string path;
+
+    ulong to_status;
+
+    SysTime time;
+
+    string rationale;
+
+    ulong kind;
+
+    Language lang;
```

Comment:
Hmm actually, I think you need to save the mutant here, the text. Because when it is lost we can't recreate it. The source file has changed

Reply:
Oh, that is absolutely true. I will fix it later today.